Makefile

```diff
@@ -13,6 +13,8 @@ local-devel-upgrade: kubectl-ctx-kind
 ## uninstall helm chart
 uninstall-local-devel: kubectl-ctx-kind
 	helm uninstall hollow-sandbox
+	# incase the crdb pvc is stuck in terminating
+	# kubectl patch pvc db -p '{"metadata":{"finalizers":null}}'
```

Comment:
maybe this should be a different target? `make force-uninstall`?

Reply:
or we could go with `make install` and  `make clean` ?

Reply:
This is now `make install` `make upgrade` and `make clean`